templates/definition/tariff/victron-forecast.yaml

```
@@ -0,0 +1,47 @@
+template: forecast-victron
+products:
+  - brand: Victron-VRM

```

This seems to be the most common term people and Victron itself use to refer to this feature.

```suggestion
  - brand: Victron
    description:
      generic: VRM Solar Forecast
```